Cargo.toml

```diff
@@ -31,7 +31,6 @@ sodiumoxide = "~0.0.9"
 temp_utp = "~0.7.0"
```

Comment:
> We have a few deps listed here without `~` - could that be fixed too please?

This sounds like work for another PR. I filled an issue and will work on that soon: https://github.com/maidsafe/crust/issues/550